.gitmodules

```diff
@@ -148,6 +148,7 @@
 [submodule "vendor/grammars/SublimeEthereum"]
 	path = vendor/grammars/SublimeEthereum
 	url = https://github.com/davidhq/SublimeEthereum.git
+	branch = linguist
```

Comment:
Locks us to the linguist branch as per https://github.com/github/linguist/pull/3973#issuecomment-461932742

Reply:
Could you pull in the just-merged changes from #4459? This was supposed to be included with #4445 but I screwed up. 😭